app/containers/App/index.js

```diff
@@ -24,24 +26,26 @@ const AppWrapper = styled.div`
   flex-direction: column;
 `;
 
-export function App(props) {
-  return (
-    <AppWrapper>
-      <Helmet
-        titleTemplate="%s - React.js Boilerplate"
-        defaultTitle="React.js Boilerplate"
-      >
-        <meta name="description" content="A React.js Boilerplate application" />
-      </Helmet>
-      <Header />
-      {React.Children.toArray(props.children)}
-      <Footer />
-    </AppWrapper>
-  );
+export class App extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  render() {
+    return (
+      <AppWrapper>
+        <Helmet
+          titleTemplate="%s - React.js Boilerplate"
+          defaultTitle="React.js Boilerplate"
+        >
+          <meta name="description" content="A React.js Boilerplate application" />
+        </Helmet>
+        <Header />
+        <Switch>
```

Comment:
That's probably because you use `connect` with your `App` container and bump into 'update blocking' issue. In this repro case: https://codesandbox.io/s/G6nWE3X0r try to press on some links and then press a back button, you'll see this bug, now remove `connect(mapStateToProps)(` from `App.js` and it will disappear. The example app does not suffer from it because we don't need to get anything from the store in `App`. I'd rather prefer to improve the docs and recommend to use [`ConnectedSwitch`](https://github.com/ReactTraining/react-router/issues/5072#issuecomment-310184271) instead of regular `Switch` while `react-router-redux` is still in beta.